gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/AbstractServiceStubSettingsClassComposer.java

```diff
@@ -149,6 +149,23 @@ public abstract class AbstractServiceStubSettingsClassComposer implements ClassC
 
   private final TransportContext transportContext;
 
+  // Maps of package->methodName to types for pagination with max_results field.
+  private static final ImmutableMap<String, ImmutableMap<String, TypeNode>>
+      PAGINATE_MAX_RESULT_TYPES =
+          ImmutableMap.of(
+              "com.google.cloud.bigquery.v2",
+              ImmutableMap.of(
```

Comment:
I guess there are some legacy RPCs use `max_results` field and are not of wrapper types?